spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -124,15 +124,12 @@ $ java -jar spring-cloud-dataflow-server-local-{project-version}.jar --maven.loc
 --maven.proxy.auth.password=proxypass1
 ----
 
-By default, the protocol is set to `http`. You can omit the auth properties if the proxy doesn't need a username and password.
+By default, the protocol is set to `http`. You can omit the auth properties if the proxy doesn't need a username and password. Also, the maven `localRepository` is set to `${user.home}/.m2/repository/` by default.
+Like in the above example, the remote repositories can be specified along with their authentication (if needed). If the remote repositories are behind a proxy, then the proxy properties can be specified as above.
 
-By default, the maven `localRepository` is set to `${user.home}/.m2/repository/`,
-and `https://repo.spring.io/libs-snapshot` will be the only remote repository. Like in the above example, the remote
-repositories can be specified along with their authentication (if needed). If the remote repositories are behind a proxy,
-then the proxy properties can be specified as above.
-
-If you want to pass these properties as environment properties, then you need to use `SPRING_APPLICATION_JSON` to set
-these properties and pass `SPRING_APPLICATION_JSON` as environment variable as below:
+As these are Spring Boot `@ConfigurationProperties` you can also specify them as environment variables, e.g. MAVEN_REMOTE_REPOSITORIES_REPO1_URL.
+Another common option, is to properties using the `SPRING_APPLICATION_JSON` environment variable.
```

Comment:
replace `is to properties with `is to set the properties`  I can change on merge if you agree.